README.md

```diff
@@ -68,6 +68,7 @@ sudo apt update
 sudo apt install golang-go nodejs npm build-essential
 ```
 
+After installing npm run `npm run build`.
```

Comment:
that isn't needed because it's part of the `make build-dependencies` but maybe you can mention, that the latest nodejs lts should be installed from the webpage https://nodejs.org/en/

Reply:
Ok, I swapped that. Dropped the npm build part and added a note to use the latest node lts.